receiver/hostmetricsreceiver/internal/scraper/loadscraper/config.go

```diff
@@ -19,4 +19,7 @@ import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostm
 // Config relating to Load Metric Scraper.
 type Config struct {
 	internal.ConfigSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
+
+	// If true, metrics will be average load per cpu
+	CPUAverage bool `mapstructure:"cpu_average"`
```

Comment:
Would it make sense to have this as a `*bool` instead, so that you can differentiate between "unset" and "false"?

Reply:
Isn't that generally done when zero values are valid but the default should be nonzero? May make more sense to wait until a validation/default path is added that requires it being unset since the underlying yaml content would remain the same.

Reply:
I also don't see a reason for having `unset` state in this config. It should be `false` by default

Reply:
> Isn't that generally done when zero values are valid but the default should be nonzero?

True, I had a Kubernetes case in mind, where the config would be persisted, so that one wouldn't know whether the persisted value was the zero value, or an explicitly set value. Given we are reading the config during the boot, we can safely change the default in the future and still know whether it was explicitly set.